### PR TITLE
Change the size of docker images

### PR DESCRIPTION
**Pull request by danwashable**
Resolves ceros/ceros-product#3944

This changes are regarding the review of the Dockerfiles in order to reduce the size of the image of ShareJS

Deploy notes:

No need to deploy.

Testing Steps:

To test it you should deploy ShareJS in the docker environment and check to see if everything is working.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,6 +7,9 @@ COPY . /usr/lib/node_modules/share
 
 RUN chmod +x /usr/lib/node_modules/share/sharejs_tests.sh
 
+WORKDIR /usr/lib/node_modules/share
```
Comment (danwashable) [inline Dockerfile]: The workdir is already set to this above, is it needed again?

File: Dockerfile [hunk 2]
```diff
@@ -7,6 +7,9 @@ COPY . /usr/lib/node_modules/share
...
+RUN npm install
```
Comment (danwashable) [inline Dockerfile]: Is this what got the tests running correctly?

**Comment by danwashable** [comment]
@tmaior Got two comments on here for you

File: Dockerfile (hunk 1 above)
Comment (tmaior) [reply Dockerfile]: Bad commit. Fixed.

File: Dockerfile (hunk 2 above)
Comment (tmaior) [reply Dockerfile]: Not really. I just cleaned. It's the right way to do it, I think. 
The testes were fixed by the changes in the sharejs_tests.sh in the Jenkins-server story

**Comment by tmaior** [comment]
Back to you @danwashable

File: Dockerfile (hunk 2 above)
Comment (danwashable) [reply Dockerfile]: Gotcha, just curious

**Comment by danwashable** [comment]
Code 👍

**Pull request merged**